samples/notebooks/python/3-semantic-function-inline.ipynb

```diff
@@ -155,9 +89,9 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "function_config = sk.SemanticFunctionConfig(prompt_config, prompt_template)\n",
-    "\n",
-    "summary_function = kernel.register_semantic_function(\"MySkill\", \"Summary\", function_config)"
+    "summarize = kernel.create_semantic_function(\"\"\"{{$input}}\n",
```

Comment:
Nit: I'd put the prompt inside a `prompt` variable and then pass that to the `create_semantic_function(prompt, ...)`